### PR TITLE
Add GSoC 2020 Wrap-up Post

### PR DESCRIPTION
**Pull request by ghalestrilo**
Adds [my GSoC wrap-up post](https://github.com/processing/p5.js/blob/main/contributor_docs/project_wrapups/ghalestrilo_gsoc_2020.md) to the contrubutor_docs folder.

### CODE REVIEW

**Comment by catarak** [comment]
@ghalestrilo I just added some suggested edits to the Google Doc you sent me—I don't know if you'd like to add them here!

**Comment by ghalestrilo** [comment]
Definitely! I just added them, please have a look again

File: contributor_docs/project_wrapups/ghalestrilo_gsoc_2020.md [hunk 1]
```diff
@@ -3,38 +3,33 @@
...
-
-I'm [Ghales](https://ghales.top), and the story above happened to me ~~(I never sent the screenshot)~~. My project for Google Summer of Code '20 was to design and implement a mobile user interface for the editor, so that everyone can use the editor comfortably wherever they are. This requires:
+The [p5.js web editor](https://github.com/processing/p5.js-web-editor) is an important tool in the p5.js ecosystem. Beginners and experts both rely on the tool for prototyping p5.js sketches because of its convenience and accessibility. However, one of its limitations is that it doesn’t work well on small-screen devices, such as smartphones. Perhaps you only have access to a smartphone, or from my personal experience, maybe you have a sketch idea while away from home or you want to show your favorite one to a friend. I wanted to create a version of the web editor that everyone could use comfortably wherever they are. I’m [Ghales](https://ghales.top), and my project for Google Summer of Code '20 was to design and implement a mobile user interface for the editor. This required:
```
Comment (catarak) [inline contributor_docs/project_wrapups/ghalestrilo_gsoc_2020.md:6]: Change "p5.js web editor" to "p5.js Web Editor"

File: contributor_docs/project_wrapups/ghalestrilo_gsoc_2020.md [hunk 2]
```diff
@@ -3,38 +3,33 @@
...
 
-Seeing how other mobile editors organize functionality helps a **lot** when thinking of where things go, because most problems already have well-tested solutions. For instance: a bottom bar with editor actions, the project and filename on the header, an explorer sidebar, etc... 
+Researching how other mobile editors organize functionality also helped a lot when thinking of where to place elements in the design, because many problems already have well-tested solutions. For example creating a bottom bar with editor actions that are used often and easy to reach with a user’s thumb, or having the project and filename in the header so it’s clear to a user what they are working on.
```
Comment (catarak) [inline contributor_docs/project_wrapups/ghalestrilo_gsoc_2020.md:17]: Add a comma after "For example", i.e. "For example, creating a bottom bar..."

File: contributor_docs/project_wrapups/ghalestrilo_gsoc_2020.md [hunk 3]
```diff
@@ -3,38 +3,33 @@
...
 
 ## Step 2: Feature Mapping
 
-  
+The next phase of work was mapping out what the current editor does onto a spreadsheet, creating what is called a  feature map. When looking at the map, notice how the UI is described in a tree structure, where the further from the root, the more clicks a functionality takes. The most urgent ones need to be as left as possible, near the user.
```
Comment (catarak) [inline contributor_docs/project_wrapups/ghalestrilo_gsoc_2020.md:21]: There are two spaces in front of "feature map" and there should just be one. Also, "feature map" should be italicized, "_feature map_"

**Comment by ghalestrilo** [comment]
Sure! I made the changes you suggested

**Comment by ghalestrilo** [comment]
@lmccart I don't have write access. can I ask you to merge this? :)

**Pull request merged**